project/src/main/java/caselab/service/attribute/AttributeService.java

```diff
@@ -56,13 +52,13 @@ public void deleteAttribute(Long id) {
         if (attributeRepository.existsById(id)) {
             attributeRepository.deleteById(id);
         } else {
-            throw new NoSuchElementException(
-                getAttributeNotFoundMessage(id)
-            );
+            throw attributeNotFound(id);
         }
     }
 
-    public String getAttributeNotFoundMessage(Long id) {
-        return messageSource.getMessage("attribute.not.found", new Object[] {id}, Locale.getDefault());
+    private EntityNotFoundException attributeNotFound(Long id) {
+        return new EntityNotFoundException(
+            messageSource.getMessage("attribute.not.found", new Object[] {id}, Locale.getDefault())
```

Comment:
![image](https://github.com/user-attachments/assets/d8317950-ac65-4e7a-94e2-dd0d8d51930a)
```suggestion
            messageSource.getMessage("attribute.not.found", new Object[] {id}, "attribute.not.found", Locale.getDefault())
```
Это чтобы NullPointerException, который бросает Objects.requireNonNull нигде вылетал